pkg/utils/event_sender.go

```diff
@@ -59,7 +62,11 @@ func GetHTTPReq(funcName string, funcPort int, kafkaTopic, namespace, eventNames
 	req.Header.Add("event-id", eventID)
 	req.Header.Add("event-time", timestamp.String())
```

Comment:
I think I was misunderstood. What I had in mind is to use the message timestamp instead of the current time
```suggestion
	req.Header.Add("event-time", kafkaMessage.Timestamp.UTC().Format(time.RFC3339))
```

Reply:
@sepetrov re `event-id` and `event-time`: removing or changing the values would be changing the meaning of those fields and represent a breaking change

Reply:
by adding the more relevant fields and new headers, we keep backwards compat but start putting in more useful info

Reply:
That is true. My suggestions are BC-breaking, because I personally to see any value in current implementation. That is @andresmgot  call again. I would be happy to have the new metadata available regardless the name.

Reply:
yes, let's avoid breaking changes if possible. Some functions may rely in the current format.